semantic.cabal

```diff
@@ -274,7 +274,7 @@ library
   autogen-modules:     Paths_semantic
   other-modules:       Paths_semantic
   build-depends:       base >= 4.12 && < 5
-                     , ansi-terminal ^>= 0.8.2
+                     , ansi-terminal >= 0.8.2 && <1
```

Comment:
Pourquoi?

Reply:
Not technically needed, but ansi-terminal just hit 0.9.0, and the old `hspec-expectations-pretty-diff` was complaining about it.

Reply:
Gotcha.